### ISSUE TITLE
Update issue in 4.0.0

### ISSUE THREAD

**Issue by Kedar-27** [airbnb/lottie-ios#1830]
@calda Tried to update the package via pods and spm. But facing error.

<img width="752" alt="Screenshot 2022-12-07 at 1 03 38 AM" src="https://user-images.githubusercontent.com/30876876/206005284-4acb166a-cfd9-4e23-a5fa-72b98cf3ddad.png">

**Comment by calda** [comment]
@Kedar-27, what version of cocoapods are you using? you can check with `pod --version`

**Comment by Kedar-27** [comment]
> @Kedar-27, what version of cocoapods are you using? you can check with `pod --version`

1.11.3

**Comment by Kedar-27** [comment]
> I'm not able to reproduce this issue on my end. Could you share a sample project and command that demonstrates this issue?
> 
> Another question: what version of git are you using? you can check with `git --version`.

git version 2.37.1 (Apple Git-137.1)

**Comment by calda** [comment]
Does the issue go away if you install [Git LFS](https://git-lfs.github.com/) by running `brew install git-lfs`?

We started using Git LFS recently. My understanding is that this is supposed to be optional (you can still clone the repo without this installed), but it may cause issues in certain circumstances.

**Comment by Kedar-27** [comment]
Yes this worked. Thanks. Checked with both pods and via spm

**Comment by AF-cgi** [comment]
But why is it necessary to install Git LFS manually to migrate to Lottie 4.0? That's not a good approach.

**Comment by calda** [comment]
This is not intentional -- my understanding was that it shouldn't be necessary for consumers to install Git LFS to clone the Lottie repo. e.g. `git clone git@github.com:airbnb/lottie-ios` works for me even without Git LFS installed.

If a lot of consumers are encountering this issue, we may need to revert the changes that adopted Git LFS.

**Comment by chrishawkinsnz** [comment]
I am having a similar issue on trying to update to 4.0.0 as well. I _do_ have git-lfs installed but it is getting tripped up on some "missing objects" when I try to update the library using SPM (via a Tuist script)
```
Creating working copy for https://github.com/airbnb/lottie-ios.git
error: Couldn’t check out revision ‘57fd1ec26f5cb00a8e5082f2b22ef300bda66ed5’:
    Downloading Tests/__Snapshots__/SnapshotTests/testAutomaticRenderingEngine.9squares_AlBoardman-0.png (35 KB)
    Error downloading object: Tests/__Snapshots__/SnapshotTests/testAutomaticRenderingEngine.9squares_AlBoardman-0.png (5431138): Smudge error: Error downloading Tests/__Snapshots__/SnapshotTests/testAutomaticRenderingEngine.9squares_AlBoardman-0.png (5431138e3d867ca8e54096c00fbd5ee2463a87f969fe7e58fab8c0efec5cb8f7): error transferring "5431138e3d867ca8e54096c00fbd5ee2463a87f969fe7e58fab8c0efec5cb8f7": [0] remote missing object 5431138e3d867ca8e54096c00fbd5ee2463a87f969fe7e58fab8c0efec5cb8f7
    Errors logged to '/Users/chawkins/alt_trademe/Tuist/Dependencies/SwiftPackageManager/.build/checkouts/lottie-ios/.git/lfs/logs/20221207T150515.729583.log'.
    Use `git lfs logs last` to view the log.
    error: external filter 'git-lfs filter-process' failed
    fatal: Tests/__Snapshots__/SnapshotTests/testAutomaticRenderingEngine.9squares_AlBoardman-0.png: smudge filter lfs failed
```

I managed to force my way through this by temporarily setting the following in my .gitconfig
```
[lfs]
     SkipDownloadErrors = true
```

**Comment by alperkayabasi** [comment]
Same issue here.

**Comment by calda** [comment]
Released 4.0.1 with a fix for this issue. Please let us know if you're still experiencing this issue with 4.0.1.

**Comment by chrishawkinsnz** [comment]
Yahoo, was just able to install 4.0.1 through SPM with no issues. Congrats on turning round a fix for this so quickly!
